opentelemetry-sdk/src/opentelemetry/sdk/trace/propagation/jaeger_format.py

```diff
@@ -0,0 +1,129 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+import urllib.parse
+
+import opentelemetry.trace as trace
+from opentelemetry import baggage
+from opentelemetry.context import Context, get_current
+from opentelemetry.trace.propagation.textmap import (
+    Getter,
+    Setter,
+    TextMapPropagator,
+    TextMapPropagatorT,
+)
+
+
+class JaegerFormat(TextMapPropagator):
```

Comment:
I know we have a B3Format, but I think it makes more sense to name this `JaegerPropagator`.

Reply:
you are right, I thought about it initially but wanted to maintain consistency. I have changed the class name and the file name, might be a good idea to rename the B3 class as well.